services/ui-src/src/components/pages/Dashboard/DashboardTable.tsx

```diff
@@ -70,16 +73,18 @@ export const DashboardTable = ({
           >
             {entering && reportId == report.id ? (
               <Spinner size="md" />
-            ) : isStateLevelUser ? (
-              "Edit"
-            ) : (
+            ) : report.status === "Approved" ||
+              report.status === "Submitted" ||
+              isAdmin ? (
               "View"
+            ) : (
+              "Edit"
```

Comment:
I logged in as help desk user and the text "Edit" shows up for in progress reports 🤔

Reply:
oooh i wasn't aware there was a helpdesk user - I'll verify what a helpdesk user should see

Reply:
helpdesk users should be view-only

Reply:
fixed, helpdesk will see view instead of edit now